lib/netsuite/response.rb

```diff
@@ -7,6 +7,8 @@ def initialize(attributes = {})
       @header   = attributes[:header]
       @body     = attributes[:body]
       @errors   = attributes[:errors] || []
+
+      raise_on_response_errors
```

Comment:
I think we should use `raise_on_response_errors!` to indicate that an exception is raised.